src/pages/about.js

```diff
@@ -9,13 +9,19 @@ export default function AboutPage() {
       description="Meet the people behind Thyck Corgis."
     >
       <div className="container">
-        <p className="title slide-in-left"> Through Thyck and Thin</p>
+        <p className="title text--center slide-in-left">
+          {" "}
```

Comment:
always watch out for these autoformatted spaces 😤 prettier making code not-so-pretty with this bs